packages/theming/theme-tokens/src/getTokens.ios.ts

```diff
@@ -1,9 +1,24 @@
+import iOSLightAliasTokens from '@fluentui-react-native/design-tokens-ios/light/tokens-aliases.json';
+import iOSDarkAliasTokens from '@fluentui-react-native/design-tokens-ios/dark/tokens-aliases.json';
 import iOSLightShadowTokens from '@fluentui-react-native/design-tokens-ios/light/tokens-shadow.json';
 import iOSDarkShadowTokens from '@fluentui-react-native/design-tokens-ios/dark/tokens-shadow.json';
+
 import { AppearanceOptions } from '@fluentui-react-native/theme-types';
 import { assertNever } from 'assert-never';
 
-export function getiOSShadowTokens(mode: AppearanceOptions) {
+export function getAliasTokens(mode: AppearanceOptions) {
+  if (mode === 'light') {
+    return iOSLightAliasTokens;
+  } else if (mode === 'dark') {
+    return iOSDarkAliasTokens;
+  } else if (mode === 'highContrast') {
+    throw new Error('highContrast is not a valid AppearanceOptions on iOS');
```

Comment:
can we also give comment to documentation why?

Reply:
Adding quick notes for some context
I did a quick google search, and it looks like RN Core is missing the ability to look that up: https://reactnative.dev/docs/accessibilityinfo
The only way _right now_ to do it is via PlatformColor: https://reactnative.dev/docs/dynamiccolorios
I imagine @mischreiber 's work of exposing trait collection may work into feeding this option into RN Core.

Reply:
Yes iOS currently doesn't have the ability to look it up, also we don't have high contrast tokens for iOS. Currently just light/dark mode, and dark elevated coming up. Ended up changing this to return the light mode tokens - needed because the default theme will always try to create a high contrast theme, and with this change it'll try to create a high contrast theme with iOS tokens. More context here: https://github.com/microsoft/fluentui-react-native/issues/2492